### DISCUSSION TITLE
[simple question] how to give access for log dir ?

### DISCUSSION THREAD

**Discussion by itweb62** [halfgaar/FlashMQ#44] [General]
Hello,

when i try to give access for flashmq user i didn't gain the access with chown flashmq:flashmq & chmod..

File '/var/log/flashmq/flashmq.log' is not there and can't be created, because '/var/log/flashmq' is also not writable

I'm not a linux expert so..

#Edit (not for this case)
For mTLS i also add SSL_CTX_use_certificate_chain_file in listener.cpp

**Comment by halfgaar** [comment]
Did you make FlashMQ run as user `flashmq`? Otherwise it should likely be `root`. And I assume you're in Docker? Then take care of proper volume permissions too.

**Reply by itweb62** [reply]
Thank you. i gonna read for docker volume permission.

and for mTLS wih SSL_CTX_use_certificate_chain_file  can i add ca_root for client authentication during handshake ? 

Best regards

**Reply by halfgaar** [reply]
TLS client authentication is on my list. The certificate needs to be validated and the username needs to be read from the 'common name' field. I haven't looked at the details.
